etna/transforms/feature_importance.py

```diff
@@ -94,3 +100,108 @@ def fit(self, df: pd.DataFrame) -> "TreeFeatureSelectionTransform":
         weights = self._get_regressors_weights(df)
         self.selected_regressors = self._select_top_k_regressors(weights, self.top_k)
         return self
+
+
+class MRMRFeatureSelectionTransform(Transform):
```

Comment:
we have a base class https://github.com/tinkoff-ai/etna/blob/master/etna/transforms/feature_selection.py#L9 now that implements `_get_regressor` and `transform` methods